### ISSUE TITLE
[TEST-FAILURE] queue.ClientQueueTest timeout

### ISSUE THREAD

**Issue by bilalyasar** [hazelcast/hazelcast#5227]
```
06:01:08 Running com.hazelcast.client.queue.ClientQueueTest
07:03:37 Build timed out (after 100 minutes). Marking the build as aborted.
```
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.6/536/console

**Comment by bilalyasar** [comment]
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.6/539/console

**Comment by sancar** [comment]
closed via #5296
